aws/src/test/java/org/apache/iceberg/aws/glue/TestGlueCatalogInteg.java

```diff
@@ -0,0 +1,271 @@
+ * under the License.
+ */
+
+package org.apache.iceberg.aws.glue;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.iceberg.AssertHelpers;
+import org.apache.iceberg.BaseMetastoreTableOperations;
+import org.apache.iceberg.DataFile;
+import org.apache.iceberg.DataFiles;
+import org.apache.iceberg.PartitionSpec;
+import org.apache.iceberg.Schema;
+import org.apache.iceberg.Table;
+import org.apache.iceberg.TableMetadata;
+import org.apache.iceberg.TableOperations;
+import org.apache.iceberg.catalog.Namespace;
+import org.apache.iceberg.catalog.TableIdentifier;
+import org.apache.iceberg.exceptions.AlreadyExistsException;
+import org.apache.iceberg.exceptions.NoSuchNamespaceException;
+import org.apache.iceberg.exceptions.NoSuchTableException;
+import org.apache.iceberg.relocated.com.google.common.collect.Sets;
+import org.apache.iceberg.types.Types;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Test;
+import software.amazon.awssdk.services.glue.GlueClient;
+import software.amazon.awssdk.services.glue.model.Database;
+import software.amazon.awssdk.services.glue.model.DeleteDatabaseRequest;
+import software.amazon.awssdk.services.glue.model.EntityNotFoundException;
+import software.amazon.awssdk.services.glue.model.GetDatabaseRequest;
+import software.amazon.awssdk.services.glue.model.GetTableRequest;
+import software.amazon.awssdk.services.glue.model.GetTableResponse;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.model.Delete;
+import software.amazon.awssdk.services.s3.model.DeleteObjectsRequest;
+import software.amazon.awssdk.services.s3.model.HeadObjectRequest;
+import software.amazon.awssdk.services.s3.model.ListObjectsV2Request;
+import software.amazon.awssdk.services.s3.model.ListObjectsV2Response;
+import software.amazon.awssdk.services.s3.model.ObjectIdentifier;
+
+@Ignore
+public class TestGlueCatalogInteg {
+
+  private static final List<String> NAMESPACES = new ArrayList<>();
+  private static final GlueClient GLUE = GlueClient.create();
+  private static final S3Client S3 = S3Client.create();
+  private static final String BUCKET = System.getenv("GLUE_BUCKET");
+  private static GlueCatalog glueCatalog;
+
+  @BeforeClass
+  public static void beforeClass() {
+    Configuration configuration = new Configuration();
+    // Iceberg uses Hadoop 2.7.3, and the hadoop-aws module does not support reading environment variable yet
+    configuration.set("fs.s3a.access.key", System.getenv("AWS_ACCESS_KEY_ID"));
+    configuration.set("fs.s3a.secret.key", System.getenv("AWS_SECRET_ACCESS_KEY"));
+    glueCatalog = new GlueCatalog(configuration, GLUE, null, "s3a://" + BUCKET);
```

Comment:
Not a problem for now but wondering if we want to add a path/folder with random name to the warehouse location inside bucket to allow concurrent runs

Reply:
yeah that's a good idea, let me do that